server/services/nextPayment.service.js

```diff
@@ -241,3 +241,19 @@ export const sendReminder = async () => {
     { $unwind: '$propertyInfo' },
   ]);
 };
+
+export const cronjob = async () => {
+  const today = new Date();
+
+  const unResolvedNextPayments = await NextPayment.aggregate([
+    { $match: { resolved: false } },
+    { $match: { expiresOn: { $lte: today } } },
+  ]);
+
+  await unResolvedNextPayments.forEach(async (nextPayment) => {
+    await resolvePendingPayment(nextPayment._id);
+    await generateNextPaymentDate({ offerId: nextPayment.offerId });
+  });
+
+  return unResolvedNextPayments.length;
```

Comment:
Let's return the unresolvedNextPayments instead of the length so that if we run this manually we get also get the unresolved fields.

```suggestion
  return unResolvedNextPayments;
```